assets/src/stories-editor/components/block-navigation/item.js

```diff
@@ -85,11 +85,11 @@ class BlockNavigationItem extends Component {
 
 	render() {
 		const { block, getBlockIndex, isSelected, onClick } = this.props;
-		const isCallToActionBlock = 'amp/amp-story-cta' === block.name;
+		const isUnMovableBlock = 'amp/amp-story-cta' === block.name || 'amp/amp-story-page-attachment' === block.name;
```

Comment:
Since `BlockNavigationList` is already doing this check, we could just pass down the information.